addons/mrp/i18n/vi.po

```diff
@@ -824,7 +824,7 @@ msgstr ""
 #: model:ir.actions.act_window,name:mrp.mrp_bom_form_action
 #: model:ir.ui.menu,name:mrp.menu_mrp_bom_form_action
 msgid "Bills of Materials"
-msgstr "Định mức nguyên liệu"
+msgstr "Định mức Nguyên liệu"
```

Comment:
Bills of Material hay BoM hay BOM là một thuật ngữ trong Sản xuất. Do đó cần thống nhất khi dịch thuật ngữ này trên toàn hệ thống. Chỗ thì Định mức nguyên liệu, chỗ thì Định mức Nguyên vật liệu, chỗ thì Định mức Vật liệu???